### DISCUSSION TITLE
[@types/jasmine] `jasmine.stringContaining()` Shouldn't Allow a `RegEx` Argument

### DISCUSSION THREAD

**Discussion by SlurpTheo** [DefinitelyTyped/DefinitelyTyped#66605] [Issues with a @types package]
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/dd5d393b861993826b55112956ded73d07c97820/types/jasmine/index.d.ts#L389

https://jasmine.github.io/api/edge/jasmine.html:
```
(static) stringContaining(expected)
Get an AsymmetricEqualityTester, usable in any matcher that uses Jasmine's equality (e.g. toEqual, toContain, or toHaveBeenCalledWith), that will succeed if the actual value is a String that contains the specified String.

Parameters:
Name	Type	Description
expected	String	
Since:
3.10.0
```

[a comment by an automated account is not shown]

**Comment by chivesrs** [comment]
This might be one of those scenarios where the implementation of the method means regexp works (NOTE: I don't actually know if this is true).

You're welcome to send out a PR that changes the typedef to exactly match the documentation, and that should fix this issue!

**Comment by Semigradsky** [comment]
PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66924
